Library/Homebrew/utils/shared_audits.rb

```diff
@@ -12,9 +12,17 @@ module SharedAudits
   def self.eol_data(product, cycle)
     @eol_data ||= T.let({}, T.nilable(T::Hash[String, T.untyped]))
     @eol_data["#{product}/#{cycle}"] ||= begin
-      result = Utils::Curl.curl_output("--location", "https://endoflife.date/api/#{product}/#{cycle}.json")
-      json = JSON.parse(result.stdout) if result.status.success?
-      json = nil if json&.dig("message")&.include?("Product not found")
+      result = Utils::Curl.curl_output("--location", "https://endoflife.date/api/v1/products/#{product}/releases/#{cycle}")
+
+      if result.status.success?
+        begin
+          json = JSON.parse(result.stdout)
+          json = nil if json&.dig("message")&.include?("Product not found")
```

Comment:
This can be never happen. They removed 404.json and now it will not be json.

- https://github.com/endoflife-date/endoflife.date/pull/2080/files#diff-6e6db31acc12e2feb1ce488d68d8ad68a0b80e86005b2f47aeb20d97ea3eb41b

Reply:
Thanks for confirming. I removed this in the latest push.